src/components/Contacts/ContactDetails/ContactDetailsHeader/ContactHeaderSection/ContactHeaderStatusSection.tsx

```diff
@@ -34,72 +32,14 @@ export const ContactHeaderStatusSection = ({
       </ContactHeaderSection>
     );
   } else if (status) {
-    let statusText = '';
-
-    switch (status) {
-      case 'NEVER_CONTACTED':
-        statusText = t('Never Contacted');
-        break;
-
-      case 'ASK_IN_FUTURE':
-        statusText = t('Ask in Future');
-        break;
-
-      case 'CULTIVATE_RELATIONSHIP':
-        statusText = t('Cultivate Relationship');
-        break;
-
-      case 'CONTACT_FOR_APPOINTMENT':
-        statusText = t('Contact for Appointment');
-        break;
-
-      case 'APPOINTMENT_SCHEDULED':
-        statusText = t('Appointment Scheduled');
-        break;
-
-      case 'CALL_FOR_DECISION':
-        statusText = t('Call for Decision');
-        break;
-
-      case 'PARTNER_FINANCIAL':
-        statusText = t('Partner - Financial');
-        break;
-
-      case 'PARTNER_SPECIAL':
-        statusText = t('Partner - Special');
-        break;
-
-      case 'PARTNER_PRAY':
-        statusText = t('Partner - Prayer');
-        break;
-
-      case 'NOT_INTERESTED':
-        statusText = t('Not Interested');
-        break;
-
-      case 'UNRESPONSIVE':
-        statusText = t('Unresponsive');
-        break;
-
-      case 'NEVER_ASK':
-        statusText = t('Never Ask');
-        break;
-
-      case 'RESEARCH_ABANDONED':
-        statusText = t('Research Abandoned');
-        break;
-
-      case 'EXPIRED_REFERRAL':
-        statusText = t('Expired Referral');
-        break;
-    }
+    const statusText = contactPartnershipStatus[status];
```

Comment:
So I'm somewhat confused by this. The ```translation.json``` file contains all of the statuses, so couldn't we just pass the ```status``` to the translation function? We do it in other places and it appears to work just fine. Unless that is incorrect and needs to be changed but it seems much simpler to do than having an additional helper function for translating the status.